data_handling.py

```diff
@@ -0,0 +1,69 @@
+def identify_missing_dates(df):
+    #Returns the total number of missing dates
+    #Returns all records with missing dates
+    pass
+
+
+#Strategy : [zeroes, drop, fill]
+def handle_missing_dates(df, strategy='drop'):
+    pass
+
+  
+def identify_missing_features(df, baseline_features=None):
+    #baseline_features is a list of given features which helps idenitify missing data in other columns
+    #returns all records with missing features in one or more columns
+    pass
+
+
+#Strategy: [drop, next, previous, average, groupby_average: day, weekend, weekdays]
+def handle_missing_features(df, strategy = 'avg'):
```

Comment:
feature should be an argument as well right?

Reply:
This function is meant to handle missing data for all of the non-timestamp data types in user's dataset. Imagine if there are more than 50 features present in my dataset.
I can have a new function to handle missing data for a given column.